vispy/app/backends/_tk.py

```diff
@@ -0,0 +1,781 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+vispy backend for Tkinter.
+"""
+
+from __future__ import division
+
+from time import sleep
+
+from ..base import (BaseApplicationBackend, BaseCanvasBackend,
+                    BaseTimerBackend)
+from ...util import keys
+from ...util.ptime import time
+
+
+# -------------------------------------------------------------------- init ---
+
+# Import
+try:
+    import sys
+    import os
+
+    _tk_on_linux = sys.platform.startswith('linux')
+    _tk_on_darwin = sys.platform == 'darwin'
+    _tk_on_windows = sys.platform.startswith('win')
+
+    try:
+        import tkinter as tk  # Python >= 3
+    except ModuleNotFoundError:
```

Comment:
@djhoese Haven't we removed all python2.7 compatibility code recently? Maybe we can just remove this here?

Reply:
I wasn't sure if Python 2.x should still be supported, if not I'll remove the distiction.

Reply:
Good point. @Wosser1sProductions?

Reply:
@djhoese So should I remove it then?

Reply:
Python 2 is no longer supported by vispy, so yeah remove any python 2 specific code. Thanks.